discoverx/explorer.py

```diff
@@ -197,6 +198,39 @@ def map(self, f) -> list[any]:
 
         return res
 
+    def map_chunked(self, f: Callable, tables_per_chunk: int, **kwargs) -> list[any]:
```

Comment:
```suggestion
    def map_chunked(self, f: Callable, tables_per_chunk: int, **kwargs) -> list[Any]:
```
any is a function, not a type